### ISSUE TITLE
Could we move the Kata Containers Architecture Committee meeting to one hour earlier?

### ISSUE THREAD

**Issue by fidencio** [kata-containers/community#266]
Currently the Kata Containers A/C meeting has been happening at 3PM UTC.
However, we've noticed some conflicts for members of the Kata Containers' Architecture Committee.

With this in mind, could we consider holding the meeting at 2PM UTC, at least while the DST doesn't hit us again?

**Comment by fidencio** [comment]
2PM UTC would translate in 7AM for folks in the west coast, if my calculations are correct.

**Comment by fengwang666** [comment]
@fidencio I prefer 3PM UTC but 2PM is also doable :)

**Comment by GabyCT** [comment]
@fidencio for be it works both 2PM or 3PM UTC

**Comment by jodh-intel** [comment]
I know it isn't ideal, but as a reminder for those that cannot attend a meeting, they are recorded by OIF and the link to the recording is added into the agenda after the meeting. See for example the "Meeting Recording:" links in:

- https://etherpad.opendev.org/p/Kata_Containers_2022_Architecture_Committee_Mtgs

**Comment by egernst** [comment]
Yeah, for me, I already miss the other calls in person that are (much) earlier, and I don't want to miss this one part of the time also. That was my main hesitancy.

**Comment by chavafg** [comment]
both works for me :)
